test/omega/test.lean

```diff
@@ -367,3 +367,9 @@ example {a b : Nat} (_ : a + b = 9) : (min a b) % 2 + (max a b) % 2 = 1 := by
 
 example {a : Int} (_ : a < if a ≤ b then a else b) : False := by omega
 example {a b : Int} : (if a < b then a else b - 1) ≤ b := by omega
+
+-- Check that we use local values.
+example (i j : Nat) (p : i ≥ j) : True := by
+  let l := j - 1
+  have _ : i ≥ l := by omega
+  trivial
```

Comment:
Could you add another test that shows that it can handle more indirection? Maybe something like
```
example (i j : Nat) (p : i ≥ j) : True := by
  let l := j - 1
  let k := l
  have _ : i ≥ k := by omega
  trivial
```

Reply:
Done!